src/send.js

```diff
@@ -1,21 +1,13 @@
-import { createSignMessage, createSignature } from "./signature"
+/* eslint-env browser */
+
+import { createSignMessage, createSignature } from './signature'
 
 const DEFAULT_GAS_PRICE = [{ amount: (2.5e-8).toFixed(9), denom: `uatom` }]
 
-export default async function send({ gas, gasPrices = DEFAULT_GAS_PRICE, memo = `` }, messages, signer, cosmosRESTURL, chainId, accountNumber, sequence) {
-  // sign transaction
-  const stdTx = createStdTx({ gas, gasPrices, memo }, messages)
-  const signMessage = createSignMessage(stdTx, { sequence, accountNumber, chainId })
-  let signature, publicKey
-  try {
-    ({ signature, publicKey } = await signer(signMessage))
-  } catch (err) {
-    throw new Error("Signing failed: " + err.message)
-  }
+export default async function send ({ gas, gasPrices = DEFAULT_GAS_PRICE, memo = `` }, messages, signer, cosmosRESTURL, chainId, accountNumber, sequence) {
+  const signedTx = createSignedTransaction({ gas, gasPrices: DEFAULT_GAS_PRICE, memo: `` }, messages, signer, chainId, accountNumber, sequence)
```

Comment:
@faboweb createSignedTransaction should be `await`'d. 

`const signedTx = await createSignedTransaction(...`

Have fixed this in my local development branch.